From 041d67e4a35c90a4201110bf2b436c390a94ba23 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Tue, 11 Apr 2023 00:19:05 +0200
Subject: [PATCH 441/529] ARM: dts: stm32: enable arm-wdt on STM32MP1 boards

Enable the arm_wdt on STM32MP1x boards. Set default timeout
to 32 seconds.

Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: Iea2b010955f68a5f1cce146e06c0ccf4b14b247c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/299848
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm/boot/dts/stm32mp135f-dk.dts   | 5 +++++
 arch/arm/boot/dts/stm32mp157a-ed1.dts  | 5 +++++
 arch/arm/boot/dts/stm32mp157c-ed1.dts  | 5 +++++
 arch/arm/boot/dts/stm32mp157d-ed1.dts  | 5 +++++
 arch/arm/boot/dts/stm32mp157f-ed1.dts  | 5 +++++
 arch/arm/boot/dts/stm32mp15xx-dkx.dtsi | 5 +++++
 6 files changed, 30 insertions(+)

--- a/arch/arm/boot/dts/stm32mp135f-dk.dts
+++ b/arch/arm/boot/dts/stm32mp135f-dk.dts
@@ -186,6 +186,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157a-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ed1.dts
@@ -144,6 +144,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &crc1 {
 	status = "okay";
 };
--- a/arch/arm/boot/dts/stm32mp157c-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157c-ed1.dts
@@ -144,6 +144,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &cpu0 {
 	cpu-supply = <&vddcore>;
 };
--- a/arch/arm/boot/dts/stm32mp157d-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ed1.dts
@@ -144,6 +144,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &cpu0 {
 	cpu-supply = <&vddcore>;
 };
--- a/arch/arm/boot/dts/stm32mp157f-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157f-ed1.dts
@@ -144,6 +144,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &cpu0 {
 	cpu-supply = <&vddcore>;
 };
--- a/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
+++ b/arch/arm/boot/dts/stm32mp15xx-dkx.dtsi
@@ -143,6 +143,11 @@
 	};
 };
 
+&arm_wdt {
+	timeout-sec = <32>;
+	status = "okay";
+};
+
 &cec {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&cec_pins_b>;
